### PR TITLE
data : Rimpyyadav

### PR DESCRIPTION
**Pull request by Rimpyyadav**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7000"></a>

### CODE REVIEW

File: data/Rimpyyadav.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Rimpy Yadav",
+  "type": "personal",
+  "bio": "Finding my better place in tech as Newbie :) Aspiring Backend webdev , Love to do Shuffle "
```
Comment (khushalarora) [inline data/Rimpyyadav.json:4]: Please add a **'Comma'** at the end of this line :+1: 
```suggestion
  "bio": "Finding my better place in tech as Newbie :) Aspiring Backend webdev , Love to do Shuffle ",
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @Rimpyyadav, thank you for contributing :tada:

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by Rimpyyadav** [comment]
Huge Thanks to Eddie and Sara for all your Support !!!

**Pull request merged**